kmp-maps/src/iosMain/kotlin/com/swmansion/kmpmaps/Map.kt

```diff
@@ -50,6 +51,7 @@ public actual fun Map(
     val polygonStyles = remember { mutableMapOf<MKPolygon, MapPolygon>() }
     val polylineStyles = remember { mutableMapOf<MKPolyline, MapPolyline>() }
     val markerMapping = remember { mutableMapOf<MKPointAnnotation, MapMarker>() }
+    val isDarkModeEnabled = isSystemInDarkTheme()
```

Comment:
I would try to call this lazily, if mapTheme is fixed we don't have to check system theme, aren't we?